Building himmelblau fails on NixOS

**Description**
Building himmelblau fails on NixOS

**Steps to Reproduce**
1. nix build github:himmelblau-idm/himmelblau  --no-write-lock-file



**Logs and Output**
Please attach relevant logs. Make sure to include outputs from the systemd journal by running:
```bash
[~]$ nix build github:himmelblau-idm/himmelblau  --no-write-lock-file
warning: not writing modified lock file of flake 'github:himmelblau-idm/himmelblau':
• Added input 'rust-overlay':
    'github:oxalica/rust-overlay/72161c6c53f6e3f8dadaf54b2204a5094c6a16ae?narHash=sha256-LX3TCDBeNpCWTDXtGyRASVcLmRPChSli34bgHnZ1DCw%3D' (2025-10-22)
• Added input 'rust-overlay/nixpkgs':
    follows 'nixpkgs'
error: builder for '/nix/store/ipnfb2zi11jmlh931ji9bblk0hiyqllq-himmelblau-2.0.0.drv' failed with exit code 101;
       last 25 log lines:
       >    Compiling systemd-journal-logger v2.2.2
       >    Compiling sd-notify v0.4.5
       >    Compiling pam_himmelblau v2.0.0 (/build/source/src/pam)
       >    Compiling selinux v2.0.0 (/build/source/src/selinux)
       >    Compiling libnss v0.9.0
       >    Compiling clap_complete v4.5.59
       >    Compiling sshd-config v2.0.0 (/build/source/src/sshd-config)
       >    Compiling qr-greeter v2.0.0 (/build/source/src/qr-greeter)
       > error: failed to run custom build command for `selinux v2.0.0 (/build/source/src/selinux)`
       >
       > Caused by:
       >   process didn't exit successfully: `/build/source/target/release/build/selinux-2b09de21a7e169ca/build-script-build` (exit status: 101)
       >   --- stdout
       >   cargo:rerun-if-changed=src/himmelblaud.te
       >   cargo:rerun-if-changed=src/himmelblaud.fc
       >   cargo:rerun-if-env-changed=HIMMELBLAU_ALLOW_MISSING_SELINUX
       >
       >   --- stderr
       >   make: /usr/share/selinux/devel/Makefile: No such file or directory
       >   make: *** No rule to make target '/usr/share/selinux/devel/Makefile'.  Stop.
       >
       >   thread 'main' panicked at src/selinux/build.rs:62:13:
       >   SELinux policy build failed
       >   note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
       > warning: build failed, waiting for other jobs to finish...
       For full logs, run:
         nix log /nix/store/ipnfb2zi11jmlh931ji9bblk0hiyqllq-himmelblau-2.0.0.drv
[~]$
```


**Environment**
- **Linux Distro**: NixOS
- **Package source (distro package/github release/self built)**: github main
- **Himmelblau Version**: 2.0.0

--- a/flake.nix
+++ b/flake.nix
@@ -22,7 +22,7 @@
           };
           cargoToml = builtins.fromTOML (builtins.readFile ./Cargo.toml);
           recipe = {
-            lib, withSelinux ? true,
+            lib, withSelinux ? false,
           }: rustPlatform.buildRustPackage {
             pname = "himmelblau";
             version = cargoToml.workspace.package.version;

